.env

```diff
@@ -0,0 +1,5 @@
+AF_DB_URL="http://localhost:3001"
```

Comment:
dont put this .env file version in PR. i shared this only for local testing

Reply:
ok will remove it